kie-server-parent/kie-server-tests/pom.xml

```diff
@@ -635,6 +637,11 @@
             <artifactId>jaxb-impl</artifactId>
             <version>${version.com.sun.xml.bind}</version>
           </dependency>
+          <dependency>
+            <groupId>org.apache.activemq</groupId>
+            <artifactId>artemis-jms-client</artifactId>
+            <version>${version.org.apache.activemq.wildfly10}</version>
+          </dependency>
```

Comment:
I am not sure I understand why is this change needed. Is it related to the uberfire->appformer rename?

Reply:
When I consolidated the dependency management in this PR, I overrode the versions of several `org.apache.activemq` dependencies, but not all the ones used by some of the kie-server tests. Other dependencies had their versions managed from wildfly-parent.

TLDR: this change is necessary to fix missing method exceptions in some of the tests.

Reply:
OK, thanks for the explanation.